libraries/rush-lib/src/cli/actions/InitAction.ts

```diff
@@ -185,7 +186,7 @@ export class InitAction extends BaseConfiglessRushAction {
       'rush.json'
     ];
 
-    const experimentalTemplateFilePaths: string[] = ['subspaces.json'];
+    const experimentalTemplateFilePaths: string[] = [`common/config/rush/${RushConstants.subspacesFilename}`];
```

Comment:
Feels like the `common/config/rush` part should also be in a const (or consts?) somewhere.

Reply:
That string appears throughout this file. Seems like it's probably out of scope for this PR.